modules/conflict_resolver/templates/menu_resolved_conflicts.tpl

```diff
@@ -50,7 +50,7 @@
                             <div class="visible-xs col-xs-12"> </div>
                             <div class="visible-xs col-xs-12"> </div>
                             <div class="col-sm-5 col-xs-12">
-                                <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href={$baseurl}/conflict_resolver/?submenu=resolved_conflicts&reset=true'">
+                                <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href='?submenu=resolved_conflicts&reset=true'">
```

Comment:
The old one got a typo. So it looks good, but it didn't work. 
You are right, after add {$baseurl}/conflict_resolver/ , it looks more clearly.